node_test/rpc_test.go

```diff
@@ -33,23 +33,6 @@ func simpleOutcome(a, b types.Address, aBalance, bBalance uint) outcome.Exit {
 	return testdata.Outcomes.Create(a, b, aBalance, bBalance, types.Address{})
 }
 
-func createLogger(logDestination *os.File, clientName *types.Address, rpcRole string) zerolog.Logger {
-	return logging.WithAddress(zerolog.New(logDestination).
-		Level(zerolog.TraceLevel).
-		With().
-		Timestamp().
-		Str("rpc", rpcRole), clientName).
```

Comment:
Now that the RPC client/server no longer accept a logger, we can't easily add log attribute (like "rpc") in the test. Instead the RPC client/server would have to add the "rpc" attribute when it sets up it's own sub-logger off the `DefaultLogger`.

I think this is an OK tradeoff for reducing the complexity of having to pass loggers around.